--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -38,7 +38,7 @@ include(cmake/gtest.cmake)
 
 # Download CEF of specified version for current platform.
 # Specify the CEF distribution version.
-set(CEF_VERSION "135.0.17+gcbc1c5b+chromium-135.0.7049.52")
+# set(CEF_VERSION "135.0.17+gcbc1c5b+chromium-135.0.7049.52") # DISABLED FOR FLATPAK
 
 # CEF_DISTRIBUTION_TYPE default value should be different based on the type of build
 # to match the official CEF releases (which don't binary-compatible between distributions).
@@ -108,8 +108,8 @@ elseif(APPLE)
 endif()
 
 message(STATUS "Using CEF for ${CMAKE_SYSTEM_NAME} - ${CEF_PLATFORM} (${CEF_ARCH_DETECTION})")
-list(APPEND CMAKE_MODULE_PATH "${CEF_ROOT}/cmake")
-find_package(CEF REQUIRED)
+# list(APPEND CMAKE_MODULE_PATH "${CEF_ROOT}/cmake") # DISABLED FOR FLATPAK
+# find_package(CEF REQUIRED) # DISABLED FOR FLATPAK
 
 
 # Visual Studio project
@@ -128,27 +128,27 @@ find_package(CEF REQUIRED)
 
 
 # Set compiler flags for CEF.
-list(REMOVE_ITEM CEF_CXX_COMPILER_FLAGS -fno-exceptions)
-list(REMOVE_ITEM CEF_CXX_COMPILER_FLAGS -fno-rtti)
-list(REMOVE_ITEM CEF_CXX_COMPILER_FLAGS -std=c++17)
-list(REMOVE_ITEM CEF_COMPILER_FLAGS -fvisibility=hidden)
-list(REMOVE_ITEM CEF_COMPILER_FLAGS -fvisibility-inlines-hidden)
-list(APPEND CEF_CXX_COMPILER_FLAGS -std=c++20)
-list(APPEND CEF_COMPILER_FLAGS -rdynamic)
-list(APPEND CEF_EXE_LINKER_FLAGS -rdynamic)
+# list(REMOVE_ITEM CEF_CXX_COMPILER_FLAGS -fno-exceptions) # DISABLED FOR FLATPAK
+# list(REMOVE_ITEM CEF_CXX_COMPILER_FLAGS -fno-rtti) # DISABLED FOR FLATPAK
+# list(REMOVE_ITEM CEF_CXX_COMPILER_FLAGS -std=c++17) # DISABLED FOR FLATPAK
+# list(REMOVE_ITEM CEF_COMPILER_FLAGS -fvisibility=hidden) # DISABLED FOR FLATPAK
+# list(REMOVE_ITEM CEF_COMPILER_FLAGS -fvisibility-inlines-hidden) # DISABLED FOR FLATPAK
+# list(APPEND CEF_CXX_COMPILER_FLAGS -std=c++20) # DISABLED FOR FLATPAK
+# list(APPEND CEF_COMPILER_FLAGS -rdynamic) # DISABLED FOR FLATPAK
+# list(APPEND CEF_EXE_LINKER_FLAGS -rdynamic) # DISABLED FOR FLATPAK
 
 if(USE_SANDBOX)
     message(STATUS "Using CEF Sandbox")
     list(APPEND CEF_COMPILER_FLAGS -DUSE_SANDBOX)
 endif()
 
-list(REMOVE_ITEM CEF_BINARY_FILES libvulkan.so.1)
+# list(REMOVE_ITEM CEF_BINARY_FILES libvulkan.so.1) # DISABLED FOR FLATPAK
 
 #
 # Include CEF's CMakeLists.txt.
 #
-include_directories(${CEF_INCLUDE_PATH})
-add_subdirectory(${CEF_LIBCEF_DLL_WRAPPER_PATH} libcef_dll_wrapper)
+# include_directories(${CEF_INCLUDE_PATH}) # DISABLED FOR FLATPAK
+# add_subdirectory(${CEF_LIBCEF_DLL_WRAPPER_PATH} libcef_dll_wrapper) # DISABLED FOR FLATPAK
 
 
 #
@@ -255,7 +255,7 @@ add_subdirectory(${CEF_LIBCEF_DLL_WRAPPER_PATH} libcef_dll_wrapper)
 #
 # Include CEF's copy of cef_sandbox.lib.
 #
-include_directories(${CEF_INCLUDE_PATH})
+# include_directories(${CEF_INCLUDE_PATH}) # DISABLED FOR FLATPAK
 
 # Disable all the configuration for linux wallpaper engine
 # When building on CI environments like Github Actions, there might be
